CHANGELOG.md

```diff
@@ -1,3 +1,10 @@
+## 0.7.x
+
+### 0.7.0
```

Comment:
Just an FYI that we can auto-generate this now based on commit log if we use tag-release.....but that means not updating changelog and package.json version head of time as part of PR.